docs/book/src/clusterctl/configuration.md

```diff
@@ -158,6 +158,7 @@ directory in the clusterctl config file as
 ```yaml
 overridesFolder: /Users/foobar/workspace/dev-releases
 ```
+Overrides only provide file replacements; instead, provider version resolution is based only on the actual repository structure.
```

Comment:
What about wrapping this note with 
```suggestion
<aside class="note warning">

<h1> Warning! </h1>

Overrides only provide file replacements; instead, provider version resolution is based only on the actual repository structure.

</aside>
```
to make it slightly more evident?

Reply:
okay i ll update it .